doc/ref_cert/RC2/index.rst

```diff
@@ -1,56 +1,10 @@
-Reference Conformance - Kubernetes based platform for Telecom
-=============================================================
-
-This is the Kubernetes Based Reference Conformance (RC-2)
-
-Release Information
```

Comment:
Don't we need this information somewhere?

Reply:
this data is in conf.py. see the PDF rendering